core/includes/update.inc

```diff
@@ -1,10 +1,10 @@
 <?php
 /**
  * @file
- * Backdrop database update API.
+ * Backdrop site update API.
  *
- * This file contains functions to perform database updates for a Backdrop
- * installation. It is included and used extensively by update.php.
+ * This file contains functions to perform database and/or config updates for a
```

Comment:
*and* is sufficient, in this case. *functions to perform database and config updates* does not mean each function perfoms database **and** config updates.

Reply:
So either is correct. Happy to stick with "and."